packages/wrapped-keys-lit-actions/src/index.ts

```diff
@@ -0,0 +1,39 @@
+import * as exportPrivateKey from './generated/common/exportPrivateKey';
+import * as generateEncryptedEthereumPrivateKey from './generated/ethereum/generateEncryptedEthereumPrivateKey';
+import * as signMessageWithEthereumEncryptedKey from './generated/ethereum/signMessageWithEthereumEncryptedKey';
+import * as signTransactionWithEthereumEncryptedKey from './generated/ethereum/signTransactionWithEthereumEncryptedKey';
+import * as generateEncryptedSolanaPrivateKey from './generated/solana/generateEncryptedSolanaPrivateKey';
+import * as signMessageWithSolanaEncryptedKey from './generated/solana/signMessageWithSolanaEncryptedKey';
+import * as signTransactionWithSolanaEncryptedKey from './generated/solana/signTransactionWithSolanaEncryptedKey';
```

Comment:
Files generated at build time, those are ignored by git so we avoid doubling the source of truth or unnecessary updates